recipes/rundec/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  name: "rundec"
+  version: "0.5"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/DavidMStraub/rundec-python/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: ffcbcecbd61349212fced3708aaca68b4d95073be47776713461f1915c9ee86c
+  patches: add-pyproject-toml-for-build-system.patch
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - ${{ stdlib('c') }}
+    - ${{ compiler('cxx') }}
+    - swig
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+
+tests:
+  - python:
+      imports:
+        - rundec
+      pip_check: true
+
+about:
+  homepage: https://github.com/DavidMStraub/rundec-python
+  summary: 'Python wrapper around CRunDec'
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+  repository: https://github.com/DavidMStraub/rundec-python
+
+extra:
+  recipe-maintainers:
+    - DavidMStraub
```

Comment:
👋 @DavidMStraub would you like to be added as a maintainer here in the same way as you were for `ckmutil` in PR https://github.com/conda-forge/staged-recipes/pull/30189 ?